src/main/webapp/assets/tpl/apps/param.html

```diff
@@ -34,6 +34,10 @@
         <input type="checkbox" class="param-value">
         <% } %>
     </td>
+    <% } else if (sensitive == true && type == 'java.lang.String') { %>
```

Comment:
Do we really need to check for type == String, or should we mask out anything that seems to be sensitive?
As a silly example, there code be a 4-digit pin code (which someone has represented as a number).

Presumably we can just mark it as password, get the value, and then rely on our type coercions like we do everywhere else?